src/core/DownloadBox.ttslua

```diff
@@ -1,17 +1,41 @@
 function onLoad()
-  self.createButton({
+  notes = self.getGMNotes()
```

Comment:
This will be a lot easier to test if it's pulled as part of the click method, rather than just on load.  What's the reason for the move?

Reply:
Notes get grabbed up here because there are different values for e.g. position and font size which will be detected by the notes too

Reply:
Ah, I see it now.  Fair, but no reason we can't re-grab it in the click to handle both?

Reply:
Indeed, changed it to re-grab on click